examples/live-chart.vala

```diff
@@ -57,7 +57,12 @@ public class Example : Gtk.Window {
             heat.add(heat_value);
             return true;
         });
-
+		
+		Timeout.add(20000, () => {
+			chart.remove_serie(rss);
+			return false;
+		});
+		
```

Comment:
Will be a nice if we could have a button to remove that serie instead of waiting 20s